src/main/java/dev/jbang/net/TrustedSources.java

```diff
@@ -193,6 +204,24 @@ protected String getJSon(Collection<String> rules) {
 		return trustedsources;
 	}
 
+	public void addTemporary(String trust) {
+		Util.infoMsg("Adding temporary " + trust);
```

Comment:
this shows up weird..how is this different from "trust once" option ?

Reply:
This _is_ the "trust once" option, thing is that the "trust once" was literally that, it would only trust once. So if you have some code that tries to access the same URL twice, for example when retrying, then you're actually accessing the same URL multiple times... which would result in the trust question to be asked multiple times for the same URL!

This is because there was no "memory" for the "trust once" option. So with this change we will keep track of all the URLs that the user answered "1" to for the duration of the JBang execution, they will be forgotten once JBang exits.

Reply:
Ok - then let's use the same terminology. Something like:

"Trusting for this one run: URL" or similar. 

Or we rename the question to be Trusting temporarily - or Trussting for run or something similar. 

Just to reduce confusion.

Reply:
Will do 👍